types/graphql/utilities/buildASTSchema.d.ts

```diff
@@ -18,7 +24,10 @@ export function buildASTSchema(ast: DocumentNode): GraphQLSchema;
  * Given an ast node, returns its string description based on a contiguous
  * block full-line of comments preceding it.
  */
```

Comment:
Also updated comment to keep it in sync with graphql-js:
https://github.com/graphql/graphql-js/blob/master/src/utilities/buildASTSchema.js#L456-L464